src/components/molecules/timeline/timeline.test.tsx

```diff
@@ -11,9 +11,12 @@ describe('Timeline', () => {
       { name: 'event2', date: '2002-02-14' },
     ]}/>);
 
+    const firstEvent = screen.getByText('14 February 2002');
+    const secondEvent = screen.getByText('13 January 2001');
+    expect(firstEvent).toBeInTheDocument();
+    expect(secondEvent).toBeInTheDocument();
+    expect(firstEvent.compareDocumentPosition(secondEvent)).toBe(4);
```

Comment:
I will test this out locally before committing but if this test would fail before your fix in `src/components/molecules/timeline/timeline.tsx` then it's always a good idea to:

- commit the test change first
- create PR
- CI should fail
- commit the fix
- CI should pass